content/news/2024-12-12-mixxing-up-pen-and-paper.md

```diff
@@ -0,0 +1,122 @@
+
+Great soundtracks (for my use case) will have all layers put behind each other in ascending order and allow me to progress trough multiple same-length loops in a song as a scene gets more and more tense. This used to be pretty easy even before Mixxx introduced loop cues.
+
+However when the composers weren't lazy when mixing down these soundtracks for an OST release, like with Portal 2, these are the least usable files for me unfortunately.
+
+*[Play "Covenant Prayers" (Halo 5)](https://music.youtube.com/watch?v=ofWai1rsI9k)*
+
+### Multiple transitioning Loops
+This is the by far most common format of big name game soundtracks by now - and before the introduction of loop cues in 2.4.1, it was mostly unusable for me.
+
+These soundtracks consist of multiple looping themes that will fit each setting in an entire area of a game. There are ambiance bits, there are combat bits, there may be a theme for the town square. The often varying lengths of those loops made it really difficult to use for me. I mean ... still no challenge for a dedicated DJ.
+
+But a storyteller having to allocate their brain to running a pen and paper session alongside mixing? No chance.
+
+Loop Cues however fixed that for me. I can now just set up to eight loops of different sizes within a single audio file and this has made some of my least favorite music pieces to my most favorite.	
+
+*[Play "a Pirate's Life" (Assassin's Creed IV - Black Flag)](https://music.youtube.com/watch?v=AuEXQnH_jOU)*
+
+## So is it worth it?
+Honestly the prep work I put into these soundtracks so far has not always payed out as much as I would have wanted. It is a lot of fun to slice and dice video game songs and build a huge library of potential epic moments in pen and paper.
+
+However as sessions often do not work as planned, trying to always find and think of the perfect music on the fly can be quite challenging. Especially because me fiddling with music is distracting in itself.
+
+I highly recommend creating a playlist for each session and having some general purpose playlists to jump into when the players inevitably drift off. Themes for places or characters (both player and non-player) can also make a big difference and are not too difficult to keep track of.
+
+However I think a dedicated app with a lot more creature comforts and a lot less DJ-specific features would be needed to make this useful for the average DM. But I think that is fine. Because everyone has their own secret sauce for spicing up their campaigns. And mine is music.
+
+*About the YouTube Music links: I do not want to endorse that service, but there are four facts of life that made me choose this option.*  
+*1. Game Soundtracks are proprietary.*   
+*2. They are all on YouTube Music.*  
+*3. Everyone and their dog has a google account.*  
+*4. The site to get all the portal 2 music from comes from the era before https was widely used everywhere.*
```

Comment:
```suggestion
```

Reply:
I thought you could create a single commit for two suggestions like this, but I am once more confused by github.